phantom-scripts/rasterize_pdf.js

```diff
@@ -12,6 +12,23 @@ if (phantom.args.length < 2 || phantom.args.length > 3) {
 			format: 'A4',
 			orientation: 'portrait',
 			border: '1.5cm'
+			header: {
+            height: "1cm",
+            contents: phantom.callback(function(pageNum, numPages) {
+                return "<div><img src="http://files.parsetfss.com/54314910-905e-4901-af3c-56382e52a40e/tfss-89f75112-5812-4d65-8d51-0a947b2ecb71-st_logo.gif" style="text-align:left; display: block "/></div>";
+            })
```

Comment:
I think making it a parameter is a good idea as this may change client to
client.

BTW, the URL is temporary and only works on QA. We will push a lot of image
assets to S3 for this soon, once we get the report layout figured out.
On Aug 5, 2015 10:11 PM, "soby" <notifications@github.com> wrote:

> In phantom-scripts/rasterize_pdf.js
> <https://github.com/soby/html2pdf/pull/1#discussion_r36323742>:
>
> > @@ -12,6 +12,23 @@ if (phantom.args.length < 2 || phantom.args.length > 3) {
> >  			format: 'A4',
> >  			orientation: 'portrait',
> >  			border: '1.5cm'
> > +			header: {
> > +            height: "1cm",
> > +            contents: phantom.callback(function(pageNum, numPages) {
> > +                return "<div><img src="http://files.parsetfss.com/54314910-905e-4901-af3c-56382e52a40e/tfss-89f75112-5812-4d65-8d51-0a947b2ecb71-st_logo.gif" style="text-align:left; display: block "/></div>";
> > +            })
>
> Nevermind, realized I could paste it into the URL. We can't have a
> specific customer logo in all reports. Should we accept this HTML as a URL
> parameter and embed it if supplied?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/soby/html2pdf/pull/1/files#r36323742>.
>